lessons/Early-Math/commutative-property/langs/en-us/pages/discover-comm.adoc

```diff
@@ -0,0 +1,70 @@
+= Discover the Commutative Property - Part 1
+
+++++
+<style>
+  table {grid-template-rows: 3fr 1fr !important;}
+  div.circleevalsexp .value,
+  div.circleevalsexp .studentBlockAnswerFilled { min-width:unset; }
+</style>
+++++
+
+@n Analyze and evaluate the two Circles below:
```

Comment:
I understand the evaluate direction, but I am not sure what you're asking me to do with the word Analyze.